_episodes/04-building-containers.md

```diff
@@ -27,6 +27,42 @@ and then deploy the container into a production environment for execution (as yo
   <figcaption>'Singularity usage workflow' via <i>Kurtzer GM, Sochat V, Bauer MW (2017) Singularity: Scientific containers for mobility of compute. PLoS ONE 12(5): e0177459. <a href="https://doi.org/10.1371/journal.pone.0177459">https://doi.org/10.1371/journal.pone.0177459</a></i></figcaption>
 </figure>
 
+# Hello World Singularity
+
+The following recipe shows how to build a hello-world container, and run the container on your local computer.
+
+- Step 1: Open a text editor (e.g., nano, vim, or gedit in a graphical environment)
+
+  ```bash
+  nano hello-world.def
+  ```
+
+- Step 2: Include the following script in the `hello-world.def` file to define the environment
```

Comment:
Thanks a lot, @amanmdesai ! However, my impression here is this "Hello World" container fits better in the next chapter, when containers are built from definition files.

Please include this example in chapter 5 instead of here, where we focus on interactive sessions.

Reply:
Thanks @michmx. I have included this example in chapter 5 now.